vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -329,14 +337,15 @@ void ReactInstanceWin::Initialize() noexcept {
 
             LoadJSBundles();
 
-            if (m_options.DeveloperSettings.IsDevModeEnabled && State() != ReactInstanceState::HasError) {
+            if (Mso::React::ReactOptions::UseDeveloperSupport(m_options.Properties) &&
```

Comment:
>UseDeveloperSupport [](start = 42, length = 19)

Pardon me asking, but why the usage here of the static method vs the instance on m_options on line: 581

Reply:
They are the same.  -- The static method is needed for places that are working with a ReactContext instead of ReactOptions.  We could probably get rid of all the instance methods and always use the static ones, but I think the instance ones are easier to read when they can be used.